testsuite/shared/src/main/java/org/wildfly/test/security/common/elytron/AggregateSecurityRealm.java

```diff
@@ -54,6 +56,12 @@ public ModelNode getAddOperation() {
         if (authorizationRealm != null) {
             addOperation.get("authorization-realm").set(authorizationRealm);
         }
+        if (authorizationRealms != null) {
```

Comment:
Is there any reason to support `authentication-realm` and `authorization-realms` attributes at the same time?
Or why isn't here `else` statement?

Reply:
These are part of the changes from #12354

Reply:
https://github.com/wildfly/wildfly/pull/12354/files#diff-9f7fd02d4150fbf86668f5d6e9f5624dR12
> NOTE: `authorization-realm` and `authorization-realms` are mutually exclusive so exactly one of these must be specified.

By this note, in this code missing else statement.

Reply:
This is just simple utility code in the testsuite, this code is not intending to be verifying the operation - if a test was written with an invalid permutation it would quickly fail model validation and the test would fail.

Reply:
Then it is for a negative test scenario when we expect to fail.
Ok, Thank you